sdk/python/packages/flet-core/src/flet_core/textfield.py

```diff
@@ -63,6 +65,13 @@ class TextCapitalization(Enum):
     SENTENCES = "sentences"
 
 
+@dataclasses.dataclass
+class InputFilter:
+    allow: bool = field(default=True)
+    regex_string: str = field(default="[0-9]")
```

Comment:
I would remove default from `regex_string` field and require developer to explicitly specify the pattern. Defaults for `allow=True` and `replacement_string=""` look good.

Reply:
Alright, diving into that.

Reply:
What do you think of this?

```python
regex_string: str = field(default="")
```